addons/website_sale/static/src/js/website_sale.js

```diff
@@ -77,6 +77,7 @@ export const WebsiteSale = publicWidget.Widget.extend(VariantMixin, {
     destroy() {
         this._super.apply(this, arguments);
         this._cleanupZoom();
+        $('[data-bs-toggle="popover"]').popover();
```

Comment:
Since the goal is to get rid of jQuery, it should no longer be introduced ;)
Search in the new code how popovers are made now